doc/plv8.md

```diff
@@ -22,6 +22,7 @@ Implemented features are as follows.
 - Remote debugger
 - Runtime environment separation across users in the same session
 - Start-up procedure
+- Update procedure
```

Comment:
Turn this into a link like:
https://github.com/plv8/plv8/pull/204/files#diff-f7db76a4bc8765fece9d587798ae00d7R27
```
- [Update procedure](#update-procedure)
```

Add it to the `README.md` TOC too:
https://github.com/plv8/plv8/pull/204/files#diff-04c6e90faac2675aa89e2176d2eec7d8R28